Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -6,21 +6,24 @@
 #include "Engine/ChildConnection.h"
 #include "Engine/NetworkObjectList.h"
 #include "EngineGlobals.h"
-#include "EntityPipeline.h"
-#include "EntityRegistry.h"
 #include "GameFramework/GameNetworkManager.h"
 #include "Net/DataReplication.h"
 #include "Net/RepLayout.h"
 #include "SocketSubsystem.h"
-#include "SpatialActorChannel.h"
-#include "SpatialConstants.h"
-#include "SpatialInteropPipelineBlock.h"
-#include "SpatialNetConnection.h"
-#include "SpatialOS.h"
-#include "SpatialPackageMapClient.h"
-#include "SpatialPendingNetGame.h"
 
-#define ENTITY_BLUEPRINTS_FOLDER "/Game/EntityBlueprints"
+#include "Interop/Connection/SpatialConnection.h"
```

Comment:
Uber nit: We alphabetizing includes?

Reply:
That's how Valentyn decided to include them. I think we could go with that.